src/azure-cli/azure/cli/command_modules/iot/_params.py

```diff
@@ -178,10 +178,30 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
                    type=int, validator=validate_fileupload_sas_ttl,
                    help='The amount of time a SAS URI generated by IoT Hub is valid before it expires,'
                         ' between 1 and 24 hours.')
+        c.argument('fileupload_storage_identity',
+                   options_list=['--fileupload-storage-identity', '--fsi'],
+                   help="The managed identity to use for file upload authentication. Use '[system]' to "
+                        "refer to the system-assigned managed identity or a resource ID to refer to a "
+                        "user-assigned managed identity.")
         c.argument('min_tls_version', options_list=['--min-tls-version', '--mintls'],
                    type=str, help='Specify the minimum TLS version to support for this hub. Can be set to'
                                   ' "1.2" to have clients that use a TLS version below 1.2 to be rejected.')
         c.argument('tags', tags_type)
+        c.argument('identities', options_list=['--assign-identity'],
+                   nargs='*', help="Accepts system or user-assigned managed identities separated by spaces. "
+                   "Use '[system]' to refer to the system-assigned identity or a resource ID to refer to "
+                   "a user-assigned identity.")
+        c.argument('identity_role', options_list=['--role'],
+                   help="Role to assign to the hub's system-assigned managed identity")
+        c.argument('identity_scopes', options_list=['--scopes'], nargs='*',
+                   help="Space separated list of scopes to assign the role (--role) "
+                   "for the system-assigned managed identity.")
+
+    with self.argument_context('iot hub identity') as c:
+        c.argument('identities', options_list=['--identities'],
+                   nargs='*', help="Accepts system or user-assigned managed identities separated by spaces. "
+                   "Use '[system]' to refer to the system-assigned identity or a resource ID to refer to a "
+                   "user-assigned identity.")
```

Comment:
`identities` supports passing in multiple values. Should the parameter name `--assign-identity` need to be changed to plural?
By the way, could the naming of this parameter be consistent with the `identities` parameter of `iot hub identity` command?

Reply:
Technically, yes - but again, we are trying to keep consistent with other services:

![image](https://user-images.githubusercontent.com/13545962/118766968-fb107480-b831-11eb-8b5c-575c38c84fca.png)

![image](https://user-images.githubusercontent.com/13545962/118767010-095e9080-b832-11eb-83ea-6ab35470a587.png)

![image](https://user-images.githubusercontent.com/13545962/118767068-209d7e00-b832-11eb-8e75-e83c991f4874.png)

Reply:
So that is to say, for `iot hub create` we expect `--assign-identity` since that will be added to the resource create, and then `iot hub identity assign` uses `--identities` since the command `assign` is implied.

Reply:
OK